### PR TITLE
AbstractCrudObject appends =fields to URL if present

### PR DESCRIPTION
**Pull request by nilesnelson**
This Pr will grab the 'fields' parameter and append it to the relative_url when using a batch in remote_read.

This will allow for more information to be returned besides just the Id when doing a batch remote_read

@grantt @derwiki-adroll

### CODE REVIEW

**Comment by nilesnelson** [comment]
@grantt @derwiki-adroll I have been using this branch for a while and haven't run into any problems

**Comment by derwiki-adroll** [comment]
Prefer git commit messages to follow "accepted" style: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

File: facebookads/objects.py [hunk 1]
```diff
@@ -576,9 +576,13 @@ def callback_failure(response):
                 if failure is not None:
                     failure(response)
 
+            relative_url = self.get_node_path()
+            if params.get('fields'):
+                relative_url = '{}?fields={}'.format('/'.join(relative_url), params['fields']),
```
Comment (derwiki-adroll) [inline facebookads/objects.py]: Would `self.get_node_path_string()` be preferred to `'/'.join(self.get_node_path()`?

**Comment by derwiki-adroll** [comment]
I'd write the git message closer to:

AbstractCrudObject appends =fields to URL if present

File: facebookads/objects.py (hunk 1 above)
Comment (nilesnelson) [inline facebookads/objects.py]: good catch, I grabbed the join from api.py. Ill make this change

**Comment by grantt** [comment]
@nilesnelson, can't auto-merge this. Update please?

**Comment by nilesnelson** [comment]
Ill get on this, the fork just got updated.

**Comment by nilesnelson** [comment]
This should be good now

**Pull request merged**